code/modules/atmospherics/machinery/binary.dm

```diff
@@ -12,16 +9,18 @@ obj/machinery/atmospherics/binary
 	var/datum/pipe_network/network2
 
 	New()
-		..()
-		switch(dir)
-			if(NORTH)
-				initialize_directions = NORTH|SOUTH
+		SPAWN(0.5 SECONDS) // regardless of the dir passed in, this stupid thing kept chosing the same result
```

Comment:
useless

Reply:
i was having problems with binary pipes always having their initialize_direction vars set to the same number without this here

Reply:
it would cause them to act as if they were facing north/south at all times. after i put that SPAWN on that, the behavior stopped

Reply:
no, it's useless. there is nothing contained in the spawn.